### DISCUSSION TITLE
Synology SSO works in a charm

### DISCUSSION THREAD

**Discussion by vlgm** [9p4/jellyfin-plugin-sso#226] [General]
I passed some time to make jellyfin working with Synology DSM + Docker + Synology SSO.
After hours passed to find a mistake under reverse proxy (X-Forwarded-Host error), SSO is now fully functional with Open ID Connect. :+1

**Comment by k4nt87** [comment]
I’m trying to set up Jellyfin with Synology DSM + Docker + Synology SSO, but I keep getting the error:
“Error redeeming code: Forbidden / no description. Try logging in again.”
I saw your comment about fixing the X-Forwarded-Host error in the reverse proxy. Could you share what exactly the mistake was and how you resolved it?
Also, how did you get jellyfin-plugin-sso to work with Synology SSO? Any tips or specific settings that helped?

**Reply by vlgm** [reply]
Hi,

I changed my domain name, but I haven't changed all my domain name under my configs files.
That why I had X-Forwarded-Host error.

My conf is like this (based on the doc https://github.com/9p4/jellyfin-plugin-sso):

**SYNO SSO Server**

GENERAL :
Server URL : xxx.yyyyy.zzz/sso

Service -> OIDC:
https://xxx.yyyyy.zzz/sso/webman/sso/.well-known/openid-configuration

Application:
App name : Jelly
URI : https://xxx.yyyyy.zzz/sso/OID/redirect/oidcjelly
ID : xxxxxxxxxxxxxxxxxxxxxxx
secret: yyyyyyyyyyyyyyyyyyyyyy


**Jellyfin sso**

Name of OID : oidcjelly

OID endpoint : https://xxx.yyyyy.zzz/sso/webman/sso/.well-known/openid-configuration

OpenID Clinet ID :xxxxxxxxxxxxxxxxxxxxxxx
OID Secret : yyyyyyyyyyyyyyyyyyyyyy
Enabled -> OK

Regards
